quarkus.ls.ext/com.redhat.quarkus.ls/src/main/resources/com/redhat/quarkus/snippets/quarkus-java.json

```diff
@@ -46,5 +46,23 @@
 		"context": {
 			"type": "io.quarkus.test.junit.SubstrateTest"
 		}
+	},
+	"Quarkus - new integration test class": {
+		"prefix": "qitrc",
```

Comment:
It should be not `qnitc` ?

Reply:
The other snippets are `qtrc` and `qntrc`, so I wanted to stick with the same naming convention. I think it's supposed to be 
"Quarkus Integration Test Resource Class"

Reply:
Oh yes you are right, makes senses.